app/scripts/modules/core/src/function/FunctionGroupings.tsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { Application } from '@spinnaker/core';
+import { Application } from 'core';
```

Comment:
I'd like to write a rule that bans imports from a raw alias (without a subpackage), but that's trickier and can come later